### DISCUSSION TITLE
Investigate org structure possibilities

### DISCUSSION THREAD

**Discussion by binford2k** [OpenVoxProject/planning#10] [General]
We have several possibilities for the structure of this group.

1. standalone 501c3 non-profit org
2. open source collective or other similar umbrellas
3. joining the linux foundation, nivenly, or other foundation
4. potentially others

**Comment by yakatz** [comment]
# Structure: US Non-Profit

Pros:
1. We have 100% control
2. Many other companies can make donations directly to us (Github, Google, TechSoup)

Cons:
1. Paperwork
2. Confirm correct legal fit
3. We need to build our own reputation

Action Items:
1. Choose a state and agent
2. Determine expected revenue (if any)
4. Have a board meeting to nominate someone to file paperwork
5. File state incorporation and request EIN
6. File federal 501(c)(3) application
Processing time for the simple application is usually a few weeks.

## Types of US non-profits:

Which of these do we really fit in:

### 501(c)(3) - most preferential tax/donation treatment

> The exempt purposes set forth in section 501(c)(3) are charitable, religious, educational, scientific, literary, testing for public safety, fostering national or international amateur sports competition, and preventing cruelty to children or animals. The term charitable is used in its generally accepted legal sense and includes relief of the poor, the distressed, or the underprivileged; advancement of religion; advancement of education or science; erecting or maintaining public buildings, monuments, or works; lessening the burdens of government; lessening neighborhood tensions; eliminating prejudice and discrimination; defending human and civil rights secured by law; and combating community deterioration and juvenile delinquency.

501(c)(3) organizations must file a yearly federal tax return. If they have less than $50,000 "gross receipts", this is a "postcard" that says "We exist and we are aware of our legal obligations". If they have more, the form needs to list the biggest donors and recipients of funds. 990-N can be filled out by anyway, but the other version of the form usually require an accountant/tax professional. There is no penalty for filing late, but failing to file for 3 years will result in the organization losing tax-exempt status.

### Other types

[IRS Reference](https://www.irs.gov/charities-non-profits/other-non-profits/requirements-for-exemption)

| Section | Description |
| -------- | ----- |
| 501(c)(1) | Corporations Organized Under Act of Congress (including Federal Credit Unions) |
| 501(c)(2) | Title Holding Corporations for Exempt Organization |
| 501(c)(4) | Civic Leagues and Social Welfare Organizations and Local Associations of Employees |
| 501(c)(5) | Labor, Agricultural and Horticultural Organizations |
| 501(c)(6) | Business Leagues, etc. |
| 501(c)(7) | Social and Recreation Clubs |
| 501(c)(8) | Fraternal Beneficiary Societies |
| 501(c)(9) | Voluntary Employees' Beneficiary Associations |
| 501(c)(10) | Domestic Fraternal Societies |
| 501(c)(11) | Teachers' Retirement Fund Associations |
| 501(c)(12) | Benevolent Life Insurance Associations, Mutual Ditch or Irrigation Companies, Mutual or Cooperative Telephone Companies, or Like Organizations (if 85 percent or more of the organization's income consists of amounts collected from members for the sole purpose of meeting losses and expenses) |
| 501(c)(13) | Cemetery Companies (owned and operated exclusively for the benefit of their members or which are not operated for profit) |
| 501(c)(14) | State Chartered Credit Unions, Mutual Reserve Funds |
| 501(c)(15) | Mutual Insurance Companies or Associations |
| 501(c)(16) | Cooperative Organizations to Finance Crop Operations |
| 501(c)(17) | Supplemental Unemployment Benefit Trusts |
| 501(c)(18) | Employee Funded Pension Trusts (created before June 25, 1959) |
| 501(c)(19) | Veterans' Organizations |
| 501(c)(21) | Black Lung Benefit Trusts |
| 501(c)(22) | Withdrawal Liability Payment Funds |
| 501(c)(25) | Title Holding Corporations or Trusts with Multiple Parents |
| 501(c)(26) | State-Sponsored High-Risk Health Coverage Organizations |
| 501(c)(27) | State-Sponsored Worker's Compensation Reinsurance Organizations |
| 501(c)(28) | National Railroad Retirement Investment Trust (45 U.S.C. 231n(j) |
| 501(c)(29) | Qualified Nonprofit Health Insurance Issuers |

## 501c3 - pretty easy to start:
1. Pick corporate officers - most states require at least 2 officers and a board of directors - and resident agent - someone who lives or works in a state and will handle any legal mail that comes in.  If we pay a resident agent, that will have a cost. I personally have experience with Maryland.
7. Most states also have a standard form for bylaws for non-profits and a small form filing fee. Most states do not have ongoing costs, but require filing annual forms. Technically, you must file with the secretary of state anywhere you do business or raise money, but this isn't observed by many small organizations.
8. Filing with the federal government:
   - If you expect to have less than $50,000 gross receipts for the next 3 years, this is a very simple form and there is a $400ish filing fee.
   - Otherwise, there is a very long form and a $800ish filing fee.
9. Keep up with tax forms every year

**Comment by yakatz** [comment]
Open Collective still requires a separate legal entity, but they handle managing funds.

**Comment by silug** [comment]
The Linux Foundation had the following advice about forks:
* Name
    * The name is important.  Come up with something distinct and trademark-friendly.
    * Ensure that we own the domain, GitHub namespace, etc. associated with the new name.  (If anyone helps by squatting on any of those, get them to transfer stuff to the new organization.)
    * Remove any existing trademark usage from the codebase.  That includes paths, command names, etc.  (While this may not be strictly necessary, it eliminates an opportunity for legal action later.)
* Governance
    * Clearly define who the maintainers of the GitHub repos (etc.).
    * Clearly define who the decision-makers are for the organization.
* Process
    * Clearly define the release management process.
    * Establish who pays for any necessary resources (CI, hosting, etc.).
* Security policy
    * Establish a security policy and a security team.
    * Work on security coordination with Perforce outside the politics involved with the fork.
* Keep it professional!
    * Avoid attacks on Perforce/Puppet.  Take the high road.

**Reply by yakatz** [reply]
Pros and cons I could think of:

Pros:
1. Established organization name lends us immediate credibility with the community - this probably isn't necessary for users, but it might matter for donors.
2. Less paperwork than doing it ourselves

Cons:
1. 501(c)(6), not 501(c)(3), so not as many donation opportunities.

Questions:
1. How do we handle things like domain registration, donations from other companies? (Probably not eligible for thinks like free Google Workspace, free Slack, etc. since it is not 501(c)(3)?)

**Comment by silug** [comment]
I'm told that the application process for setting up a non-profit entity in the US can take 18 months.

LF does not require a separate legal entity (we'd be a project under their 501(c)(6) umbrella), but they do require a LF member to sponsor/support the application.

**Reply by yakatz** [reply]
That was once true, but now it doesn't take anywhere near that long - the actual incorporation takes a few minutes. IRS approval of the 1023-EZ (for small organizations) takes no more than 2 months. The 1023 regular form (more than $50k) takes about 7 months.

**Comment by bastelfreak** [comment]
> Open Collective still requires a separate legal entity, but they handle managing funds

@yakatz  can you elaborate on that a bit?

There are a few confusing names out there. opencollective.com is the platform. https://oscollective.org/ acts as the fiscal host /legal entity for vox pupuli, via opencollective.com. But vox pupuli has no own legal entity, we are just part of oscollective.

**Reply by yakatz** [reply]
My mistake - I know I was missing something. The original comment said Open Collective, not Open Source Collective. I thought it wasn't the right thing, but I went to opencollective.org and that redirects to opencollective.com too. Since you already deal with OS Collective, can you write something about that?

**Reply by yakatz** [reply]
Similar to Linux Foundation, OS Collective is 501(c)(6)

**Comment by anarcat** [comment]
I, for the record, do not care much about the legal structure generated from this. I am much more interested by the processes used to discuss and establish things and I must say that, so far, we're doing such a better job than Perforce at this that we just can't fail.

In other words, I abstain from a vote here and delegate my vote to people that know more about US or .eu organizational matters (but do let me know if you want this in Canada, that I know a bit more ;).

**Comment by jay7x** [comment]
While I don't really care much about the legal questions too, but.. Please note, that registering an entity under US/EU laws will make some nations much harder to participate in some activities. I cannot imagine any other neutral and developed law jurisdiction though.. maybe Switzerland/Austria/Luxembourg 🤷🏻‍♂️

**Reply by alaunay** [reply]
[Liechtenstein](https://fr.wikipedia.org/wiki/Liechtenstein) ? :)

**Reply by pheeef** [reply]
Founding a non-profit in Austria is pretty simple and straight forward. I have done it before and it took us around a week.

**Reply by jay7x** [reply]
Anything which is in Europe, but not under EU law is preferred I'd say.. but I'm not sure we should care about that too much..

**Comment by anarcat** [comment]
> While I don't really care much about the legal questions too, but.. Please note, that registering an entity under US/EU laws will make some nations much harder to participate in some activities. I cannot imagine any other neutral and developed law jurisdiction though.. maybe Switzerland/Austria/Luxembourg 🤷🏻‍♂️ 

I would assume this would favor Europe instead of the US, which
typically have broader US sanctions (e.g. Cuba).

But again, I would favor the people's skillset more than those kind of
politics.

**Comment by yakatz** [comment]
This is trending towards being moot (of no further significance) since there are currently discussions to bring this under the VoxPupuli umbrella.

**Reply by bastelfreak** [reply]
Vox Pupuli is currently under the https://oscollective.org/ umbrella. But it doesn't need to stay like this. If it makes sense, we can still create whatever legal entity makes sense or join another foundation.

**Comment by thomasgoirand** [comment]
If a legal entity is formed, it can be legally attacked.  Don't do that. Also even worse if you do this in USA, which would probably be the worse possible choice.

There are many ways one can do community collaboration, and a legal entity is not really needed. For example, Debian has "trusted organizations", that are holding the Debian assets (like Debian France, Swiss, or SPI in USA).  Assets could be: money, domain names, servers, etc. But Debian itself, isn't a legal entity.

This worked for us since 1992, why wouldn't it work in this case?

**Reply by ekohl** [reply]
Forgive my ignorance but isn't such a trusted organization also a legal entity? The way I read this is that we need to figure out the legal situation and what's best for the community

**Reply by maxadamo** [reply]
This is how it works for Forgejo: [who owns Forgejo](https://forgejo.org/faq/#who-owns-the-forgejo-domains-and-trademarks). So, they are not an organisation, but they under Codeberg, which is a no-profit org based in Berlin, and they receive fundings somewhere else in the Netherlands.

**Comment by maxadamo** [comment]
NLnet provides support and funding for organizations and individuals working on open-source projects: [https://nlnet.nl/news/2024/20241201-call.html](https://nlnet.nl/news/2024/20241201-call.html)

Applying for funding with NLnet doesn’t require having a legal entity registered in the Netherlands and it does not require a legal entity at all. Individuals, research organizations, non-profits, public institutions, and companies are all eligible to apply.

But if a legal entity is needed, it's worth mentioning that starting one in the Netherlands is relatively inexpensive and straightforward.  

NLnet can donate up to _"50.000 euro — with the possibility to scale them up if there is proven potential."_

Anybody going at FOSDEM this year? NLnet will be there: [https://nlnet.nl/events/20250201/FOSDEM/index.html](https://nlnet.nl/events/20250201/FOSDEM/index.html) ... we'll also be there with our organisation.

**Reply by ekohl** [reply]
I will be at FOSDEM so I'd be happy to have a chat

**Comment by maxadamo** [comment]
You may also want to have a look into this approach: [Who Owns Forgejo?](https://forgejo.org/faq/#who-owns-the-forgejo-domains-and-trademarks).

If there’s interest and a clear need, I’m happy to explore the possibility of a similar solution, though I can’t make any promises. 

Our booth at FOSDEM is located here: [FOSDEM Stands](https://fosdem.org/2025/stands/)
**F (Level 0) → F (Level 1 - Group) → Booth No. 6**

**Comment by fifieldt** [comment]
OpenInfra foundation: https://openinfra.org/ might be an option.

**Reply by maxadamo** [reply]
as far as I can tell, VoxPupuli is already registered with OpenCollective. 
OpenCollective is in the US, Common Conservancy is based in EU and they are equal. 
I see that OpenInfra is focusing on Infrastructure projects, and they also have a branch for different projects: https://opendev.org/ 

These foundations, allow relaying funding, provide legal assistance, and are home for this type of projects. 
Maybe there should be a clear reason to switch from one to another, or to have many (if possible). 


I paste you the following from OpenInfra website: 

_OpenInfra Foundation hosts open source projects that contribute to the advancement of today’s infrastructure. From cloud hosting to the driving force behind 5G, these projects are code first and operate under the guiding principles we call the “Four Opens.” These projects include [OpenStack](https://www.openstack.org/), [Airship](https://airshipit.org/), [Kata Containers](https://katacontainers.io/), [OpenInfra Labs](https://openinfralabs.org/), [StarlingX](https://starlingx.io/), and [Zuul](https://zuul-ci.org/)._

**Reply by silug** [reply]
> OpenInfra foundation: https://openinfra.org/ might be an option.

In case you missed it: https://www.linuxfoundation.org/press/open-infrastructure-foundation-board-announces-intent-to-join-the-linux-foundation

**Comment by ripienaar** [comment]
Reminder that joining any LF related thing means irreversibly giving away all trademarks and that the project may never ever leave.
